api/web_components/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Web Components
+parent: API
+nav_order: 20
+has_children: true
+---
+# Data Commons Web Components
+
+Embed [Data Commons](https://datacommons.org)
+[statistical variable](https://datacommons.org/tools/statvar) observation
+visualizations in your web application.
+
+<img src="/assets/images/web_components/all.png" style="width: 100%; max-width: 1024px;">
+
+## Usage
+
+Include `datacommons.js` in your html's
+`<head>...</head>` tag. Then use `datacommons-*` [web
+component](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) tags
+to add embedded data visualizations.
+
+Example:
+
+```html
+<html>
+  <head>
+    <script src="https://datacommons.org/datacommons.js"></script>
+  </head>
+  <body>
+    <datacommons-line
+      header="US Population Over Time"
+      place="country/USA"
+      variables="Count_Person"
+    ></datacommons-line>
+  </body>
+</html>
+```
+
+[Playground](https://lit.dev/playground/#gist=c0c88276739f4f6061807cc943937a14) ([source](/assets/examples/web-components/line-chart.html))
+
+## Components
+
+- [datacommons-bar](/api/web_components/bar)
+- [datacommons-highlight](/api/web_components/highlight)
+- [datacommons-gauge](/api/web_components/gauge)
+- [datacommons-line](/api/web_components/line)
+- [datacommons-map](/api/web_components/map)
+- [datacommons-pie](/api/web_components/pie)
+- [datacommons-ranking](/api/web_components/ranking)
+- [datacommons-scatter](/api/web_components/scatter)
+- [datacommons-slider](/api/web_components/slider)
+
+## Examples
+
+- [Static page with all chart types playground](https://lit.dev/playground/#gist=822ce6018bb41113c866d703760c1def) ([source](/assets/examples/web-components/all-charts.html)):
+- [Dynamically updating charts playground](https://lit.dev/playground/#gist=9e3ac88e162248f849dd276ff5895ad0) ([source](/assets/examples/web-components/dynamic-map.html))
+
+## Variables and places
+
+Data Commons web components visualize statistical variables about one or more places.
+Variables and places are identified by [Data Commons Identifiers](/glossary.html#dcid), or [DCID](/glossary.html#dcid)s.
+
+To find the [DCID](/glossary.html#dcid) of a place or variable:
+
+1. Browse all 175K+ variables with the [Data Commons Statistical Variable Explorer](https://datacommons.org/tools/statvar).
```

Comment:
consider skipping the count here which would get out of date..

Reply:
Good point. Removed the count.